pkg/init/backend/interactive.go

```diff
@@ -211,12 +212,18 @@ func (o *InteractiveBackend) PersonalizeDevfileconfig(devfileobj parser.DevfileO
 func PrintConfiguration(config asker.DevfileConfiguration) {
 	color.New(color.Bold, color.FgGreen).Println("Current component configuration:")
 
-	for key, container := range config {
+	keys := make([]string, len(config))
```

Comment:
Doing this, you get a slice with n elemens already on it, and later with append you add new ones: you will finish with 2*n elements.
Either you keep this make and use keys[i] = key, or you can use `make([]string, 0, len(config))` to use append.